src/views/Pools/components/PoolCard/CardFooter/index.tsx

```diff
@@ -40,7 +41,7 @@ const Footer: React.FC<FooterProps> = ({ pool, account, isAutoVault = false }) =
       <ExpandableButtonWrapper>
         <Flex alignItems="center">
           {isAutoVault ? <CompoundingPoolTag /> : <ManualPoolTag />}
-          {tooltipVisible && tooltip}
+          {tooltipVisible && ReactDOM.createPortal(tooltip, document.getElementById('root'))}
```

Comment:
This should be `bottom` I think
![Uploading Screenshot 2021-05-21 at 13.53.39.png…]()

Reply:
done